terraform/context_plan2_test.go

```diff
@@ -346,11 +404,44 @@ resource "test_object" "a" {
 	})
 
 	p := simpleMockProvider()
+	p.GetProviderSchemaResponse = &providers.GetProviderSchemaResponse{
+		Provider: providers.Schema{Block: simpleTestSchema()},
+		ResourceTypes: map[string]providers.Schema{
+			"test_object": {
+				Block: &configschema.Block{
+					Attributes: map[string]*configschema.Attribute{
+						"arg": {Type: cty.String, Optional: true},
+					},
+				},
+			},
+		},
+	}
+	p.ReadResourceFn = func(req providers.ReadResourceRequest) (resp providers.ReadResourceResponse) {
+		panic("why am I here")
```

Comment:
why am I here? 🤔

Reply:
Heh funnily enough I woke up this morning and my first thought in bed was "oh I think I forgot to remove that `panic`" and indeed, my subconscious mind was right, it turns out!

Thanks for noting it.